File: beecrowd/1/1012/1012.java
import java.io.IOException;
import java.util.Scanner;
import java.lang.Math;

class Main {

    public static void main(String[] args) throws IOException {

        Scanner sc = new Scanner(System.in);
        double A, B, C;
        A = sc.nextDouble();
        B = sc.nextDouble();
        C = sc.nextDouble();
        System.out.printf("TRIANGULO: %.3f\n",(A*C)/2);
        System.out.printf("CIRCULO: %.3f\n",3.14159 * Math.pow(C,2));
        System.out.printf("TRAPEZIO: %.3f\n",((A+B) * C)/2);
        System.out.printf("QUADRADO: %.3f\n",B*B);
        System.out.printf("RETANGULO: %.3f\n",A*B);
        sc.close();

    }

}

File: beecrowd/1/1012/1012.cs
using System;

class URI {

    static void Main(string[] args) {
        double A, B, C;
        string[] input = Console.ReadLine().Split(' ');
        A = double.Parse(input[0]);
        B = double.Parse(input[1]);
        C = double.Parse(input[2]);
        Console.WriteLine("TRIANGULO: {0}",((A*C)/2).ToString("0.000"));
        Console.WriteLine("CIRCULO: {0}",(3.14159 * Math.Pow(C,2)).ToString("0.000"));
        Console.WriteLine("TRAPEZIO: {0}",(((A+B) * C)/2).ToString("0.000"));
        Console.WriteLine("QUADRADO: {0}",(B*B).ToString("0.000"));
        Console.WriteLine("RETANGULO: {0}",(A*B).ToString("0.000"));
    }
}
